network/wsPeer.go

```diff
@@ -209,6 +209,11 @@ type wsPeer struct {
 	// throttledOutgoingConnection determines if this outgoing connection will be throttled bassed on it's
 	// performance or not. Throttled connections are more likely to be short-lived connections.
 	throttledOutgoingConnection bool
+
+	// clientDataStore is a generic key/value store used to store client-side data entries associated with a particular peer.
+	clientDataStore map[string]interface{}
+
+	clientDataStoreMu deadlock.Mutex
```

Comment:
please add the trivial comment `clientDataStoreMu` synchronizes access to the `clientDataStore` map